packages/fxa-payments-server/server/lib/server.js

```diff
@@ -2,25 +2,25 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-'use strict';
+"use strict";
```

Comment:
I thought our prettier spec would have these all stay single quote?

Reply:
I'm a little confused what happened with the style changes too. I worry all the rebasing and conflict merging might have left things a little screwy. I should probably just re-prettify.

Reply:
I'm not clear what is being applied, but running prettier makes an even bigger diff than this, so best to hold on that.